sherpa/plot/tests/test_plot.py

```diff
@@ -115,9 +115,9 @@ def test_splitplot(self):
         # sp.addplot(rp)
 
 
+@requires_data
 class test_contour(SherpaTestCase):
```

Comment:
Other than checking the contour routines do not crash, it doesn't look like these are doing much. Can we add a comment to note this (perhaps as a TODO to indicate needs a closer look)?